charts/addons/values.yaml

```diff
@@ -48,9 +48,9 @@ aws-alb-ingress-controller:
   ## VPC ID of k8s cluster, required if ec2metadata is unavailable from controller pod
   ## Required if autoDiscoverAwsVpcID != true
   awsVpcID: "YOUR_EKS_CLUSTER_VPC_ID"
-  extraEnv:
-    AWS_ACCESS_KEY_ID: "YOUR_AWS_ACCESS_KEY_ID"
-    AWS_SECRET_ACCESS_KEY: "YOUR_AWS_SECRET_ACCESS_KEY"
+  extraEnv: {}
+    # AWS_ACCESS_KEY_ID: "YOUR_AWS_ACCESS_KEY_ID"
+    # AWS_SECRET_ACCESS_KEY: "YOUR_AWS_SECRET_ACCESS_KEY"
 
```

Comment:
Can you make the comments (AWS_ACCESS_KEY_ID: "YOUR_AWS_ACCESS_KEY_ID", AWS_SECRET_ACCESS_KEY: "YOUR_AWS_SECRET_ACCESS_KEY") inline with the code (extraEnv)

Reply:
We are using base templates from here(https://github.com/helm/charts/blob/master/incubator/aws-alb-ingress-controller/values.yaml#L39)

I think it's better to comment as the base template consists.

Reply:
Our current pega/values.yaml & addons/values.yaml comply with the default rules of yamllint which we use in our travis builds to validate values.yaml files. I see that current PR build failure is because of indentation.

Reply:
fixed the indentation warning

Reply:
@MadhuriArugula  can you please review the changes.